### DISCUSSION TITLE
How to use without compose?

### DISCUSSION THREAD

**Discussion by mny459** [patrykandpatrick/vico#329] [Questions] [answered]
**Describe the bug**
According to the doc, I add a view in xml like this

```
    <com.patrykandpatrick.vico.views.chart.ChartView
        android:id="@+id/line_chart"
        android:layout_width="0dp"
        android:layout_height="wrap_content"
        app:charts="line"
        app:layout_constraintEnd_toEndOf="parent"
        app:layout_constraintStart_toStartOf="parent"
        app:layout_constraintTop_toTopOf="@id/toolbar"
        app:showBottomAxis="true"
        app:showStartAxis="true" />
```

And add data in the code

```
mBinding.lineChart.setModel(entryModelOf(4f, 12f, 8f, 16f))
```

But nothing show

**Module**

- [ ] `compose`
- [x] `views`
- [ ] `compose-m2`
- [ ] `compose-m3`
- [x] `core` (select if none of the above apply)

If possible, could you provide a sampe with pure view system?

**Comment by patrickmichalik** [comment, marked as answer]
Hello! Please try replacing `app:charts="line"` with `app:chart="line"`. `charts` is a `ComposedChartView` attribute. If the documentation suggests otherwise, could you please let us know where this error appears? We’ll then correct it.

Regarding your request for a view-based sample chart, the sample app has eight such charts. `AndroidViewBinding` is used, but what you see in the lambdas has nothing to do with Jetpack Compose. Also, several simpler examples can be found in the wiki.

**Reply by mny459** [reply]
I am sorry, It is my fault. I didn't notice the difference in 's', It is works fine now. 
Think your answer, and this great work! Have a great day!

**Reply by patrickmichalik** [reply]
It’s all good. I’m glad everything works now. Cheers!
